projects/plugins/jetpack/modules/related-posts.php

```diff
@@ -53,7 +53,7 @@ public function action_on_load() {
 		}
 
 		// Load Customizer controls.
-		if ( class_exists( 'WP_Customize_Manager' ) ) {
+		if ( class_exists( 'WP_Customize_Manager' ) && class_exists( 'WP_Customize_Control' ) ) {
```

Comment:
Optional suggestion: Since we're touching the line anyway, we might switch to `::class` since it can make it more obvious to static analysis that these strings are supposed to be class names.
```suggestion
		if ( class_exists( WP_Customize_Manager::class ) && class_exists( WP_Customize_Control::class ) ) {
```